File: programmers/pg_lv1_최소직사각형.js
function solution(sizes) {
  let maxArr = [];
  let minArr = [];

  sizes.forEach((ele) => {
    ele.sort((a, b) => b - a);
    maxArr.push(ele[0]);
    minArr.push(ele[1]);
  });

  return Math.max(...maxArr) * Math.max(...minArr);
}

File: python/programmers/pg_lv1_최소직사각형.py
def solution(sizes):
    sorted_sizes = [sorted(i) for i in sizes]

    maxRow = 0
    maxCol = 0
    for i in sorted_sizes:
        maxRow = max(i[0], maxRow)
        maxCol = max(i[1], maxCol)
        
    return maxRow * maxCol